### PR TITLE
feat: add rename to README.md

### PR DESCRIPTION
**Pull request by barriebyron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # o1js &nbsp; [![npm version](https://img.shields.io/npm/v/o1js.svg?style=flat)](https://www.npmjs.com/package/o1js) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/o1-labs/o1js/blob/main/CONTRIBUTING.md)
 
+(O)1 Labs has renamed SnarkyJS to o1js, effective immediately!
+
+This name change reflects the evolution of our vision for the premiere toolkit used by developers to build zero knowledge-enabled applications, while paying homage to our technology's recursive proof generation capabilities.
+
+Your favorite functionalities stay the same. Transitioning to o1js a quick and easy process:
```
Comment (garwalsh) [inline README.md:7]: Your favorite functionality stays the same and transitioning to o1js a quick and easy process:

Comment (barriebyron) [inline README.md:7]:
```suggestion
Your favorite functionality stays the same and transitioning to o1js is a quick and easy process:
```

**Review by garwalsh — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
 # o1js &nbsp; [![npm version](https://img.shields.io/npm/v/o1js.svg?style=flat)](https://www.npmjs.com/package/o1js) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/o1-labs/o1js/blob/main/CONTRIBUTING.md)
 
+(O)1 Labs has renamed SnarkyJS to o1js, effective immediately!
```
Comment (mitschabaude) [inline README.md:3]:
```suggestion
O(1) Labs has renamed SnarkyJS to o1js, effective immediately!
```

**Review by mitschabaude — APPROVED** [review]

**Comment by barriebyron** [comment]
thanks @mitschabaude  for the eagle eye, I copy pasted with haste!

**Review by ymekuria — APPROVED** [review]

**Pull request merged**